pkg/minikube/node/cache.go

```diff
@@ -278,13 +286,16 @@ func imagesInConfigFile() ([]string, error) {
 	return []string{}, nil
 }
 
-func updateKicImageRepo(imgName string, repo string) string {
+func updateKicImageRepo(imgName string, repo string) (string, string) {
 	image := strings.TrimPrefix(imgName, "gcr.io/")
 	if repo == constants.AliyunMirror {
 		// for aliyun registry must strip namespace from image name, e.g.
 		//   registry.cn-hangzhou.aliyuncs.com/google_containers/k8s-minikube/kicbase:v0.0.25 will not work
 		//   registry.cn-hangzhou.aliyuncs.com/google_containers/kicbase:v0.0.25 does work
 		image = strings.TrimPrefix(image, "k8s-minikube/")
 	}
-	return path.Join(repo, image)
+	baseImg := path.Join(repo, image)
+	// try a fallback image without sha, because #11068
+	fallbackImg := strings.Split(image, "@sha256:")[0]
```

Comment:
`pkg/driver/kic/types.go` did not export the image's sha, so I removed the sha part from the repo updated image.